tests/compile.sh

```diff
@@ -56,6 +56,9 @@ set +x
 if [[ $MACHINE_ID == macosx.* ]] || [[ $MACHINE_ID == linux.* ]]; then
   source $PATHTR/modulefiles/${MACHINE_ID}/fv3
 else
+  if [[ $MACHINE_ID == wcoss2 ]]; then
+    source /apps/prod/lmodules/startLmod
```

Comment:
I guess this will eventually be moved to /work/noaa/stmp/jminsuk/rt-update/NEMS/src/conf/module-setup.*sh.inc?

Reply:
Yes. Or if they switch to Lmod version of modules, sourcing this script might not be necessary at all.